mne/report.py

```diff
@@ -46,6 +46,33 @@
 # PLOTTING FUNCTIONS
 
 
+def _scene_to_fig(fig):
+    from scipy.misc import imread
+    import matplotlib.pyplot as plt
+    mayavi = None
+    try:
```

Comment:
it's okay to have two checks. I guess what I'm saying is that this exact same pattern (importing mayavi) is repeated in the file thrice (and I suspect many other times in other files too). Calls for a function in the utils I guess?